package.json

```diff
@@ -12,13 +12,16 @@
     "type-definitions/dom-event.d.ts"
   ],
   "scripts": {
-    "build": "npm run build-dist && uglifyjs dist/mostDomEvent.js -o dist/mostDomEvent.min.js",
-    "build-dist": "mkdir -p dist && rollup -c && rollup -c rollup.es.js",
-    "lint": "jsinspect src && jsinspect test && eslint src test",
+    "build": "npm run build:dist && npm run build:min && npm run build:flow",
+    "build:dist": "mkdir -p dist && rollup -c && rollup -c rollup.es.js",
+    "build:min": "uglifyjs dist/mostDomEvent.js -o dist/mostDomEvent.min.js",
+    "build:flow": "cpy src/index.js.flow dist --rename mostDomEvent.js.flow",
```

Comment:
I think we also have to copy to `mostDomEvent.es.js.flow`, similar to [most/core](https://github.com/mostjs/core/blob/master/packages/core/package.json#L21).

Reply:
module field is supported by both rollup and webpack. There's no reasons to duplicate stuff. Flow will look at main field only.

Reply:
Oh, hmmm, I thought flow fully resolved the js filename first, and then looked for a file of the same name with `.flow` appended.  I'll do a quick test just to convince myself we don't need the "es.js.flow" copy ...

Reply:
I just found [this issue](https://github.com/facebook/flow/issues/3980), which seems to confirm.